packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -214,7 +214,7 @@ const CFDsListing = () => {
 
                     <div className='cfd-full-row'>
                         <Text line_height='m' weight='bold' color='prominent'>
-                            {localize('Other CFDs')}
+                            {localize('Other CFD Platforms')}
```

Comment:
help to update the onboarding page as well 🙏🏼

Reply:
It's already correct there
<img width="1542" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/311df8a5-722b-4e6f-ad0d-7c873e56fc9d">